src/nagini_translation/translators/call.py

```diff
@@ -149,6 +149,35 @@ def _translate_super(self, node: ast.Call, ctx: Context) -> StmtsAndExpr:
         else:
             raise InvalidProgramException(node, 'invalid.super.call')
 
+    def translate_adt_cons(self, cons: PythonClass, args: List, pos,
```

Comment:
Incomplete / missing type annotations for args, pos

Reply:
Fixed.